### ISSUE TITLE
Fail to achieve the result of paper

### ISSUE THREAD

**Issue by lee1k** [xhu248/semi_cotrast_seg#2]
Thanks for your nice work! 
When I try to run run_coseg.sh and run_seg.sh in this project, I cannot get the result like those in paper, and even worse than the baseline. How do I need to modify the source code or trainging plan to achieve the result mentioned in the paper?

**Comment by lee1k** [comment]
Functions adjust_learning_rate and warmup_learning_rate in util.py have not relate parameters, are there any affect to training if commenting these two function?

**Comment by aakash-saboo** [comment]
I am also not able to reproduce the results. I get 78% accuracy which is even lower than the "random" baseline in the paper.
@xhu248  could you please tell us which hyperparameters were used for the paper? The paper also doesn't give a full description of the hyperparameter values.

**Comment by ericKuo722** [comment]
@aakash-saboo could you please let me know how did you run the preprocessing.py? I got the error when transposing numpy.array with incorrect dimension. The code is " image = image[:, :, 0].transpose((0, 2, 1)) ". I've tried to adjust the dimension by myself but still can't get a correct one. Thank you!

**Comment by wangyihub** [comment]
@xhu248 Thank you for your contribution.While i am very intersted in the method about contrast learning proposed by your team,i have some question :
1.In your paper,said that"The training epoch for contrastive learning is 70, and for segmentation training, it stops after 120 epochs."I am not sure if the "contrastive learning" and the "segmentation training" are pointed global contrast(SimCLR) in step 1 and Local contrast(pixel contrast) in step 2.If so,i want to know how many epoches about fine-tune in step 3.
2. whether the code  " image = image[:, :, 0].transpose((0, 2, 1)) " should be removed in preprocessing.py?Because,error will be reported during code execution raised by this code,and it is not rational to appear in preprocessing.py.
3. In ablation experiment about table 1,i want to know which organ you choose to train in hippocampus dataset.In download_dataset.py,i see Task07_Pancreas is  chosed to download in default.
I am looking forward to your early reply,and thank for your contribution again!
